pkg/model/resources/nodeup.go

```diff
@@ -76,29 +92,28 @@ download-or-bust() {
 
   while true; do
     for url in "${urls[@]}"; do
-      commands=(
-        "curl -f --compressed -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10"
-        "wget --compression=auto -O "${file}" --connect-timeout=20 --tries=6 --wait=10"
-        "curl -f -Lo "${file}" --connect-timeout 20 --retry 6 --retry-delay 10"
-        "wget -O "${file}" --connect-timeout=20 --tries=6 --wait=10"
-      )
-      for cmd in "${commands[@]}"; do
-        echo "Attempting download with: ${cmd} {url}"
-        if ! (${cmd} "${url}"); then
-          echo "== Download failed with ${cmd} =="
-          continue
-        fi
-        if ! validate-hash "${file}" "${hash}"; then
-          echo "== Hash validation of ${url} failed. Retrying. =="
-          rm -f "${file}"
-        else
-          echo "== Downloaded ${url} (SHA256 = ${hash}) =="
-          return 0
-        fi
-      done
-    done
-
-    echo "All downloads failed; sleeping before retrying"
+	  local access_token=""
+	  if [[ "$GCE_ACCESS_TOKEN_URL" != "" ]]; then
```

Comment:
As @hakman pointed out, we need to `if [[ "$GCE_ACCESS_TOKEN_URL" != "" ]]; then` -> `if [[ "${GCE_ACCESS_TOKEN_URL:-}" != "" ]]; then` because of `set -o nounset`.

It's a little hard to see, but it was logged in https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kops/13449/pull-kops-e2e-kubernetes-aws/1511472037783146496/artifacts/13.233.128.192/cloud-init-output.log  You can get there by clicking on the failing test, then the artifacts tab, then artifacts / pick an IP address / cloud--init-output.log.